src/services/import/excel/XLSImporterValidateObservations.ts

```diff
@@ -95,34 +116,70 @@ export default class XLSImporterValidateObservations extends AbstractImporter {
       }
       excelData.possibleClones = excelData.addedData.length - map.size;
     } catch (e) {
-      throw new CustomError(e.message, 500);
+      throw new CustomError(e.message, 400);
     }
   };
 
-  private setXLSDataToObservation = async (_excelData: DataCheck): Promise<void> => {};
+  private setXLSDataToObservation = async (excelData: DataCheck): Promise<void> => {
+      const defaults = {
+        manipulated: 'U',
+        movedBeforeTheCapture: 9,
+        catchingMethod: 'U',
+        catchingLures: 'U',
+        accuracyOfDate: 9,
+        accuracyOfCoordinates: 0,
+        pullusAge: 99,
```

Comment:
Seems that 99 is unappropriate code for defaults, as it confirms that birds is pullus.

Reply:
All this defaults have to be  rechecked